automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspaceNotebooksPage.scala

```diff
@@ -11,18 +11,28 @@ import org.scalatest.selenium.Page
 class WorkspaceNotebooksPage(namespace: String, name: String)(implicit webDriver: WebDriver)
   extends WorkspacePage(namespace, name) with Page with PageUtil[WorkspaceNotebooksPage] {
 
-  override def awaitReady(): Unit = await enabled sparkClustersHeader
+  override def awaitReady(): Unit = {
+
+    clustersTable.awaitReady()
+  }
```

Comment:
`awaitReady()` is called upon going to a page. Does this override need to await that the table is ready || the error is present?

Reply:
Can I do `clustersTable.awaitReady() || await visible clusterErrorMessage` or will that not work?

Reply:
You have to do `await condition { thing1ready || thing2Visible }`

Reply:
ahhhh, that makes sense! Thanks

Reply:
hmm - awaitVisible and awaitReady return Unit rather than Boolean, so I can't actually use them with await condition - will have to find a different way

Reply:
I could do

```

  override def awaitReady(): Unit = {
    await condition (enabled(clustersTableContainer) || enabled(clusterErrorMessage))
  }

 private val clustersTableContainer = testId("spark-clusters-table")
  private val clustersTable = Table("spark-clusters-table")
  private val clusterErrorMessage = testId("notebooks-error")
```

Reply:
Yeah, it's gonna be `await condition { clustersTable.getState == "ready" || find(clusterErrorMessage).isDefined }` or something like that.